File: objc/KonkeSdk4YinYue_ObjcExample/ViewController.m
//
//  ViewController.m
//  KonkeSdk4YinYue_ObjcExample
//
//  Created by 吴伟城 on 2023/8/7.
//

#import "ViewController.h"
@import KonkeSdk4YinYue;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)openSDK:(id)sender {
    [KonkeSdk4YinYueRunner openSdkHomeBy:self];
}

@end

File: ios/KonkeSdk4YinYue_Example_iOS/ViewController.swift
//
//  ViewController.swift
//  KonkeSdk4YinYue_Example_iOS
//
//  Created by 吴伟城 on 2023/5/3.
//

import UIKit
import KonkeSdk4YinYue

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func openSDK(_ sender: Any) {
        KonkeSdk4YinYueRunner.openSdkHome(by: self)
    }
}

